tests/blake2b.cc

```diff
@@ -191,7 +189,72 @@ TEST_P(Blake2bKAT, TryKAT)
   }
 }
 
+TEST_P(Blake2bKAT, TryKATStreaming)
+{
+  auto test_case = GetParam();
+
+  // Skip tests with key.
+  if (test_case.key.size() != 0) {
+    return;
+  }
+
+  // Required for `hacl_vec256_support()`.
+  hacl_init_cpu_features();
+
+  {
+    bytes got_hash(64);
+
+    // Init
+    Hacl_Streaming_Blake2_blake2b_32_state_s* state =
+      Hacl_Streaming_Blake2_blake2b_32_no_key_create_in();
+    Hacl_Streaming_Blake2_blake2b_32_no_key_init(state);
+
+    // Update
+    Hacl_Streaming_Blake2_blake2b_32_no_key_update(
+      state, test_case.input.data(), test_case.input.size());
+
+    // Finish
+    Hacl_Streaming_Blake2_blake2b_32_no_key_finish(state, got_hash.data());
+    Hacl_Streaming_Blake2_blake2b_32_no_key_free(state);
+
+    EXPECT_EQ(test_case.digest, got_hash);
+  }
+
+#ifdef HACL_CAN_COMPILE_VEC256
+  if (hacl_vec256_support()) {
+    bytes got_hash(64);
+
+    // Init
+    Hacl_Streaming_Blake2b_256_blake2b_256_state* state =
```

Comment:
SEGFAULT

Reply:
This API is not exposed in any header. So nothing we should try to test. Note it as an issue and remove this.

Reply:
Thanks! 👍🏻

Reply:
I was afraid it is what I was thinking it is.
Please disable this for now. See https://github.com/project-everest/hacl-star/issues/586 for details.

Reply:
Disabled in [bb2566a](https://github.com/cryspen/hacl-packages/pull/125/commits/bb2566a992e0056eccf0756a6f57a26e0da5db62).